components/svgs/icons/website-categories/LGBT.svg

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 22.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 16 16" style="enable-background:new 0 0 16 16;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#010101;}
+	.st1{display:none;}
+	.st2{display:inline;}
+	.st3{fill:#FF0000;}
+</style>

```

Comment:
The stylesheet should not be in the svg otherwise it will break when used in the web